### PR TITLE
Bug fix, trying to get tokens to work to look at photons

### PR DESCRIPTION
**Pull request by hswanson13**

### CODE REVIEW

**Comment by hswanson13** [comment]
Maybe dont merge this but you can look at this if you want, not sure

File: Phase2TimingAnalyzer/plugins/Phase2TimingAnalyzer.cc [hunk 1]
```diff
@@ -157,11 +174,26 @@ class Phase2TimingAnalyzer : public edm::one::EDAnalyzer<edm::one::SharedResourc
...
 Phase2TimingAnalyzer::Phase2TimingAnalyzer(const edm::ParameterSet& iConfig):
+
+  //photonHLTFilterNamesFile_(iConfig.getParameter<string> ("photonHLTFilterNamesFile")),
+  //gedPhotonCoresToken_(consumes<vector<reco::PhotonCore> >(iConfig.getParameter<edm::InputTag>("gedPhotonCores"))),
+
   _jetTimingTools(consumesCollector()),
+
+  //photonCollectionToken_(consumes<std::reco::Photon>>(edm::InputTag("photons"))),
+
+  //photonCollectionToken_(consumes<std::vector<reco::Photon>>(edm::InputTag("photons"))),
```
Comment (danbarto) [inline Phase2TimingAnalyzer/plugins/Phase2TimingAnalyzer.cc]: L189 should actually work

File: Phase2TimingAnalyzer/plugins/Phase2TimingAnalyzer.cc [hunk 2]
```diff
@@ -235,9 +267,9 @@ void Phase2TimingAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSet
   int nrecojets = 0;
```
Comment (danbarto) [inline Phase2TimingAnalyzer/plugins/Phase2TimingAnalyzer.cc:260]: below L236 you'll need something like
```
Handle< std::vector<reco::Photon> > photonCollectionH;
```

and then below L246
```
iEvent.getByToken(photonCollectionToken_,photonCollectionH);
```

then you should be able to iterate through the photonCollectionH somewhere downstream

**Comment by danbarto** [comment]
Added comments on what works for me

**Pull request merged**